staff_features/shared/step_definitions.rb

```diff
@@ -65,6 +65,24 @@
   end
 end
 
+When 'the user fills in {string} with {string} and selects {string} in the {string} form' do |label, value, selected_value, form_title|
```

Comment:
This is a bit of a nitpick but `the user fills in the {string} field` would be more clear to me here

Reply:
We have intentionally left the word `field` out since it's redundant. Everything that can be filled, selected, and generally is part of a form, can be considered a field.
The same applies to the `user clicks on {string}`, which is intentionally ambiguous, as it may be a button or a link, or it may look like a button but may contain a link, or the opposite.
The same applies to the word `record` for entities of the system. We use capitalized words for all the records like: `Accession`, `Resource`, `Extent`, etc.

We want the Gherkin to be as clear, concise, and compact as possible, and not so much the step definitions, which if you look at the block arguments, is quite clear what they refer to.

If we suffix the word `field`, shouldn't we do the same for all the `{string}`s, and for all form user input definitions (selects etc.)?
In my opinion, this redundancy reduces clarity:
'the user fills in {string} `field` with {string} `value` and selects {string} `from the dropdown menu here?` in the {string} form' do |label, value, selected_value, form_title|

If you think we must apply this change, I believe we should open a new PR, to apply these changes everywhere, to avoid inconsistency between various step definitions that use the current phrasing.

Reply:
I guess thinking about it more, I'd agree that `fills in {string} with {string}` implies that it is a text field, so I can live with it. Like I said it's a bit of a nitpick so it probably wouldn't be worth working out any far-reaching changes.